kernel/thread.c

```diff
@@ -328,6 +328,7 @@ static inline int z_vrfy_k_thread_name_copy(k_tid_t thread,
 #endif /* CONFIG_USERSPACE */
 
 
+#ifdef CONFIG_MULTITHREADING
 #ifdef CONFIG_STACK_SENTINEL
```

Comment:
Can this just be `if defined(CONFIG_MULTITHREADING) && defined(CONFIG_STACK_SENTINEL)`